coap-core/src/main/java/com/mbed/coap/server/internal/CoapUdpMessaging.java

```diff
@@ -163,10 +188,6 @@ private void stopTransactionTimeoutWorker() {
 
     @Override
     public void stop0() {
-        if (duplicationDetector != null) {
-            duplicationDetector.stop();
```

Comment:
Shouldn't the cache stop be called here? In case of the default one. I don't see its stop called anywhere else than from tests.

Reply:
Fixed.